### PR TITLE
feat: up ceramic, ipfs 54

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,24 +68,26 @@
...
     "eslint": "^7.15.0",
     "eslint-config-3box": "^0.2.0",
-    "ipfs": "^0.52.3",
+    "ipfs": "^0.54.2",
     "it-all": "^1.0.2",
     "jest": "^26.6.2",
     "jest-environment-uint8array": "^1.0.0",
     "jsdoc-babel": "^0.5.0",
     "jsdoc-to-markdown": "^6.0.1",
+    "multiformats": "~4.6.1",
```
Comment (oed) [inline package.json:90]: Is this needed here now?

Comment (zachferland) [reply package.json:90]: is used in tests, nice to explicitly reference

File: test/threeid-provider.test.js [hunk 2]
```diff
@@ -160,7 +161,7 @@ describe('ThreeIdProvider', () => {
       expect(idw1.id).toEqual(idw2.id)
     })
 
-    it('Removes authSecret from keychain and creates instance', async () => {
+    it.only('Removes authSecret from keychain and creates instance', async () => {
```
Comment (zachferland) [inline test/threeid-provider.test.js:164]: @ukstv may be an interesting bug here to look at, added comments here and logs where the update is happening, this fails more often than not, although some times passes. Sometimes it fails on first update, sometimes second. Failed update just means it awaits doc change, but returns with out change being applied. 

Started to try try to replicate in a more limited context - https://github.com/zachferland/doc-change-race/blob/master/index.js but have not yet (this passes fine so far, as expected i guess)

File: src/three-idx.ts [hunk 3]
```diff
@@ -232,7 +232,9 @@ export class ThreeIDX {
       if (!this.docs[KEYCHAIN_DEF].metadata.schema) {
         update.metadata = { schema: ThreeIdKeychain }
       }
+      console.log(this.docs[KEYCHAIN_DEF].content)
```
Comment (zachferland) [inline src/three-idx.ts:235]: logging here, when change fails, these log the same values (can also log update, does exist)

**Pull request closed without merging**